plugin.py

```diff
@@ -795,10 +797,19 @@ def list_items(self) -> List[sublime.ListInputItem]:
             if os.path.isdir(os.path.join(julia_environments_path, env))
         ]
         env_paths = [os.path.join(julia_environments_path, env) for env in env_names]
-        # Add workspace folders if they are valid Julia project environments
+        # Add all environments withing a working folder.
         for folder in self.workspace_folders:
-            if folder.path not in env_paths and is_julia_environment(folder.path):
-                items.append(sublime.ListInputItem(folder.name, folder.path, kind=KIND_WORKSPACE_FOLDER))
+            for subdir, dirs, files in os.walk(folder.path):
```

Comment:
Not sure if this is a good thing to do. I'd say it should at least exclude `.git` folders, because they can contain a lot of subfolders. Maybe it would be sufficient to only check the top-level folders in each of the workspace folders? I don't know what exact directory structure you use, but this would at least include the `test` folders.

Reply:
It wasn't really tailored to my own directory structure since there isn't a fixed one. I didn't want to make too many assumptions about it. I agree that it should ignore all `dot` subfolders though. I'll add it.
Only checking top-level folders feels pretty arbitrary, I don't really think there will be the situation in which someone has so many directories that checking for the environment will cause lag to be honest.